File: 程式語言 V2答案檔/C#/209.cs
using System;

public class TQC209
{
    public static void Main(string[] args)
    {
        int input;
        input = int.Parse(Console.ReadLine());
        if (input == 0) 
            PrintHorizontalMultiplicationTable();
        else if (input == 1)
            PrintVerticalMultiplicationTable();
        else
            Console.WriteLine("error");
    }

    public static void PrintHorizontalMultiplicationTable() {
        for (int i = 1; i <= 5; i++) {
            for (int j = 1; j <= 5; j++) {
                Console.Write(String.Format("{0} x {1} = {2}\t", i, j, i * j));
            }
            Console.WriteLine("");
        }
    }

    public static void PrintVerticalMultiplicationTable() {
        for (int i = 1; i <= 5; i++) {
            for (int j = 1; j <= 5; j++) {
                Console.Write(String.Format("{0} x {1} = {2}\t", j, i, i * j));
            }
            Console.WriteLine("");
        }
    }
}

File: 程式語言 V2答案檔/Python/209.py
n = input()

if n != '0' and n != '1':
    print('error')
else:
    for i in range(1, 6):
        for j in range(1, 6):
            if n == '0':
                f = str(i) + ' x ' + str(j) + ' = ' + str(i * j)
            elif n == '1':
                f = str(j) + ' x ' + str(i) + ' = ' + str(i * j)
            if j == 5:
                print('%s\t'%f)
            else:
                print('%s\t'%f, end = '')